x-pack/legacy/plugins/uptime/public/pages/page_header.tsx

```diff
@@ -60,6 +66,11 @@ export const PageHeader = ({ headingText, breadcrumbs, datePicker = true }: Page
             <h1>{headingText}</h1>
           </EuiTitle>
         </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <Link to={`${SETTINGS_ROUTE}`}>
+            <EuiButtonEmpty data-test-subj="settings-page-link">{settingsLinkText}</EuiButtonEmpty>
+          </Link>
```

Comment:
i think we should add settings icon as well like other apps, APM etc 
![image](https://user-images.githubusercontent.com/3505601/76078071-36fb4900-5fa2-11ea-9985-9763e54d0227.png)